docs/architecture.md

```diff
@@ -31,17 +32,17 @@ Notes:
 These are made using your user's k8s permissions. It consumes your OpenShift OAuth Proxy value during the call and makes the call directly to the endpoint of the caller's choosing. This functionality eliminates CORS issues when talking to services that are in OpenShift Console.
 
 - Proxy call
-    - These are always POST calls due to the nature of how the HTTP spec works -- we want to send metadata to the endpoint and GETs are not used for this
-    - The call is made from the client with the target OpenShift Route path, request params, and the actual method you want to use
+  - These are always POST calls due to the nature of how the HTTP spec works -- we want to send metadata to the endpoint and GETs are not used for this
+  - The call is made from the client with the target OpenShift Route path, request params, and the actual method you want to use
 - K8s call
-    - Powered by the [OpenShift SDK](https://github.com/openshift/dynamic-plugin-sdk)
-    - These are crafted with the k8s<Verb>Resource calls and makes use of a known model system
-        - Each call takes a `ResourceModel` and additional params like namespace (if applicable) & name (if applicable) and the SDK crafts a url to the endpoint
-        - Our server will take this call and proxy it onto the K8s API server
-    - See [SDK tidbits](./SDK.md)
+  - Powered by the [OpenShift SDK](https://github.com/openshift/dynamic-plugin-sdk)
+  - These are crafted with the k8s<Verb>Resource calls and makes use of a known model system
```

Comment:
Nit: in some places we use k8s and in some K8s. Should we make them uniform 🤔 ?

Reply:
This particular line is referring to the SDK functions:
<img width="429" alt="image" src="https://github.com/opendatahub-io/odh-dashboard/assets/14068621/0389768a-9d64-4a2b-8c36-7b79de1b287d">

In my experience the case of the `k` would be lower case when starting a camel-case name, otherwise upper case for pascal-case or if found in the middle of the name.

Reply:
I think I referred to the wrong line here. I was referring to this document(only the text parts-unrelated to your changes-should've mentioned this).